cloud-config/run-scripts/ami-python.sh

```diff
@@ -1,79 +0,0 @@
-#!/bin/bash
```

Comment:
How is mod_wsgi installed now?

Reply:
It is installed in the AMI from ubuntu package. Not needed for python 3.6.